composer.json

```diff
@@ -9,8 +9,7 @@
     "require": {
         "php": ">=5.3.3",
         "symfony/symfony": "2.3.*",
-        "doctrine/orm": "~2.2,>=2.2.3,<2.5",
-        "doctrine/dbal": "<2.5",
+        "doctrine/orm": "~2.2.3||~2.3.0||~2.4.8||~2.5",
```

Comment:
is the doc wrong https://getcomposer.org/doc/01-basic-usage.md#package-versions ?